src/test/java/cloud/fogbow/common/util/SystemUserUtilTest.java

```diff
@@ -6,7 +6,12 @@
 import org.junit.Test;
 
 public class SystemUserUtilTest {
-    // Test if a SystemUser is correctly seralized and later deserialized
+
```

Comment:
I think SystemUserUtil, SystemUser and (more importante) SerializedEntityHolder are very complicated to understand (this is because I have problems with generics, I guess). Why not adding an static serialize and deserialize methods to the SystemUser class. "Sometimes, the elegant implementation is just a function ..."

Reply:
@ANKerD you missed adressing this. If you not agree, remember to tell so.

Reply:
An static serialize and deserialize methods to the SystemUser class seems cleaner.